bitrise/bitrise.go

```diff
@@ -355,6 +355,53 @@ func (artifact BuildArtifact) DownloadArtifact(filepath string) error {
 	return err
 }
 
+// AbortBuild ...
+func (app App) AbortBuild(buildSlug string, abortReason string) error {
+	params := make(map[string]interface{})
+	params["abort_reason"] = abortReason
+	params["abort_with_success"] = false
+	params["skip_notifications"] = true
+
+	b, err := json.Marshal(params)
```

Comment:
```suggestion

type buildAbortParams struct {
	AbortReason       string `json:"abort_reason"`
	AbortWithSucces   bool   `json:"abort_with_success"`
	SkipNotifications bool   `json:"skip_notifications"`
}

// AbortBuild ...
func (app App) AbortBuild(buildSlug string, abortReason string) error {
	b, err := json.Marshal(buildAbortParams{
		AbortReason:       abortReason,
		AbortWithSucces:   false,
		SkipNotifications: true})
```

Reply:
I tried a few different ways to get this to work with a struct thanks for the example